apps/condo/domains/acquiring/schema/MultiPayment.js

```diff
@@ -55,9 +55,8 @@ const access = require('@condo/domains/acquiring/access/MultiPayment')
 const get = require('lodash/get')
 const Big = require('big.js')
 
-
 const MultiPayment = new GQLListSchema('MultiPayment', {
-    schemaDoc: 'Information about resident\'s payment for single or multiple services/receipts',
```

Comment:
does it conflict with linter?

Reply:
No, you can inspect the code using `yarn lint --quiet`